tests/Unit/PrestaShopBundle/Entity/FeatureFlagTest.php

```diff
@@ -39,6 +39,7 @@ public function testConstructFeatureFlag()
         $featureFlag = new FeatureFlag('PrestaShop');
 
         // assert nothing, this test aims to validate we can construct the entity
+        $this->assertTrue(true);
```

Comment:
```suggestion
```

Reply:
no - this is dummy assertion to supress this test to being listed in the phpunit as as the comment says, no meaningful assertion is expected

Reply:
No, we prefer having notice than a useless line of code.

Reply:
This is not useless! Why developer should be bombarded with non usefull/expected message?

Thankfully, this is the only message for all phpunit tests and now it can be fixed.

Reply:
I'm afraid this test brings no value, the whole test can be removed. Running a test just to run a constructor without checking everything is ok is useless.

Reply:
Makes sense as constructor is run in another tests below. Code updated. Let's merge it.